services/deployment-agent/src/simcore_service_deployment_agent/subtask.py

```diff
@@ -0,0 +1,18 @@
+from abc import ABC, abstractmethod
+
+class SubTask(ABC):
+    def __init__(self, name):
+        self.name = name
+        super().__init__()
+    
+    @abstractmethod
+    async def init(self):
+        pass
+    
+    @abstractmethod
+    async def check_for_changes(self):
+        pass
+    
+    @abstractmethod
+    async def cleanup(self):
+        pass
```

Comment:
last newline missing ... there is a code setting to check for that

Reply:
done